### DISCUSSION TITLE
Can't upload more than 1Mb with Form View

### DISCUSSION THREAD

**Discussion by alprotasov** [nocodb/nocodb#1135] [Q&A] [answered]
Hello!

I want to upload 5 images in one form.
![image](https://user-images.githubusercontent.com/7497700/153137689-1b39d5d9-b46e-46d0-b28a-1b356f346522.png)
NC_REQUEST_BODY_SIZE increased.
But I get 413 Request Entity Too Large error
How can I increase it?
Thanks!

**Comment by dstala** [comment]
@alprotasov - Could you help us with more information on your setup? You can collect project-info & paste it here to start with. 
Tried multi attachment field shared form view on localhost project setup with both SQLite & MySQL database - I was able to attach large size files (>10 MB) without any issue. 

Also, AFAIK - NC_REQUEST_BODY_SIZE should not impact attachment filesize. The error you see possibly could be a limit from the database you speaking with.

**Comment by alprotasov** [comment]
Thanks for fast reply. How can I collect it?

**Comment by o1lab** [comment]
https://www.youtube.com/watch?v=AUSNN-RCwhE

**Comment by alprotasov** [comment]
Node: **v12.22.6**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **mysql2**
ProjectOnRootDB: **true**
RootDB: **mysql2**
PackageVersion: **0.84.9**

**Reply by o1lab** [reply]
possible to paste the network error from browser console ?

**Reply by alprotasov** [reply]
https://app.birdeatsbug.com/sessions/1NIW-yHwWYTsRu3DhlpDG?systemPane=active
all the info is right to a video. I'll remove the form later

**Reply by o1lab** [reply]
> network error from browser console ?

- cmd+alt+i or ctrl+alt+i
- go to network tab
- hit submit

There will be an error in red - copy paste from preview/response sub tab

**Reply by alprotasov** [reply]
There is an 413 error as I mentioned.
`<html>
<head><title>413 Request Entity Too Large</title></head>
<body>
<center><h1>413 Request Entity Too Large</h1></center>
<hr><center>nginx/1.21.3</center>
</body>
</html>





`

**Reply by alprotasov** [reply]
I guess nginx config is the reason, isn't it? https://docs.nocodb.com/getting-started/installation#docker-compose I used this method to deploy NocoDB

**Comment by alprotasov** [comment, marked as answer]
Yes, the reason is nginx, I updated the value inside nginx-proxy container (used workaround part here https://github.com/nginx-proxy/nginx-proxy/issues/690#issuecomment-594662115 ). Trying to get how to do this in docker-compose or mounted file
